src/matching/team_recommendations.py

```diff
@@ -13,127 +36,84 @@ def get_team_recommendations(email):  # GET
     Return:
         a list of recommended teams to join
     """
+    skills_weight = 1.2
+    seriousness_weight = 1.1
 
     user = coll("users").find_one({"_id": email})
     if not user:
         return {"message": "Invalid user"}, 403
 
     # basic info about users
     skills = user["skills"]
-    interests = user["interests"]
-    prizes = user["prizes"]
     seriousness = user["seriousness"]
 
-    names = set()
-    matches = []
+    all_open_teams = coll("teams").aggregate([{"$match": {"complete": False}}])
+    all_open_members = coll("users").aggregate([{"$match": {"team_id": {"$all": [all_open_teams]}}}])
+
+    team_map = dict()
+
+    # map of distances
+    for member in all_open_members:
+        team_id = member["team_id"]
+        dis = lv_distance(user, member)
+        if team_id in team_map:
```

Comment:
I think this should be `if team_id not in team_map` - might want to double check this